pgmpy/estimators/HillClimbSearch.py

```diff
@@ -158,6 +158,8 @@ def estimate(self, start=None, tabu_length=0, max_indegree=None):
 
             if best_operation is None:
```

Comment:
@karttikeya I think both the if conditions can be merged in a single statement. Something like:
```python
if best_operation is None or best_score_delta < epsilon:
    break
```